docs/eventing/samples/kafka/channel/README.md

```diff
@@ -151,3 +151,75 @@ Now you can see the events in the log of the `ksvc` using the command:
 ```
 kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container
 ```
+
+## Authentication against an Apache Kafka
```

Comment:
Authentication of what against Kafka though? Again, I still think this is confusing because underneath in the intro line we only talk about KafkaSource - I would change the heading to something like "Authenticating a `KafkaSource` object using TLS and SASL", or whatever is accurate, that states what is being authenticated by what.

Reply:
the source was a typo - I am sorry. This doc is only for the channel (Source has already docs for that, since a long time)

I will fix the intro, best I can

Reply:
This is so much better, thank you so much Matthias! <3